CIP-XXXX/README.md

```diff
@@ -0,0 +1,431 @@
+on Swagger!
+
+* [Version 1](https://app.swaggerhub.com/apis/CatastrophicCardano/FaucetAPI/1)
+
+#### Successful Responses
+
+##### Valid (200)
+
+First Successful Request
+
+```json
+{
+  "code": 200,
+  "lovelaces": "2000000",
+  "queue_position": 23,
+  "status": "accepted",
+  "tokens": {
+    "a0028f350aaabe0545fdcb56b039bfb08e4bb4d8c4d7c3c7d481c235.484f534b59": "29433292000000"
+  }
+}
+```
+
+##### Valid Queued (201)
+
+Subsequent Successful Request (Address + Code Match) prior to token distribution
+
+```json
+{
+  "code": 201,
+  "lovelaces": "2000000",
+  "queue_position": 1,
+  "status": "queued",
+  "tokens": {
+    "a0028f350aaabe0545fdcb56b039bfb08e4bb4d8c4d7c3c7d481c235.484f534b59": "29433292000000"
+  }
+}
+```
+
+##### Valid Complete (202)
+
+Subsequent Successful Request (Address + Code Match) after token(s) are distributed
+
+```json
+{
+  "code": 202,
+  "lovelaces": "2000000",
+  "status": "claimed",
+  "tokens": {
+    "a0028f350aaabe0545fdcb56b039bfb08e4bb4d8c4d7c3c7d481c235.484f534b59": "29433292000000"
+  },
+  "tx_hash": "TX1234"
+}
+```
+
+#### Error Responses
+
+##### Bad Request - Invalid Address (400)
+
+The provided address is not a valid Cardano address
+
+```json
+{
+   "code": 400,
+   "status": "invalidaddress"
+}
+```
+
+##### Bad Request - Missing Code (400)
+
+No code was provided in the request
+
+```json
+{
+   "code": 400,
+   "status": "missingcode"
+}
+```
+
+##### Bad Request - Invalid Network (400)
+
+The wallet provided is from the wrong network (testnet/mainnet)
+
+```json
+{
+   "code": 400,
+   "status": "invalidnetwork"
+}
+```
+
+##### Invalid - Not Known (404)
+
+The specified code does not exist
+
+```json
+{
+  "code": 404,
+  "status": "notfound"
+}
+```
+
+##### Invalid - Already Claimed (409)
+
+An address was already used (if not code present) or the code presented was found but the address did not match
+
+```json
+{
+  "code": 409,
+  "status": "alreadyclaimed"
+}
+```
+
+##### Invalid - Expired (410)
+
+For time-limited fountains, a code of 410 means that the period for redemption has expired
+
+```json 
+{
+  "code": 410,
+  "status": "expired"
+}
+```
+
+##### Invalid - Too Early (425)
+
+For time-limited fountains, a code of 425 means that the period for redemption has not begun yet
+
+```json
+{
+  "code": 425,
+  "status": "tooearly"
+}
+```
+
+##### Invalid - Rate Limited (429)
+
+Rate limiting settings and details are left to the discretion and implementation of individual projects. A status code
+of 429 or this status response should be considered as a rate limiting response.
+
+```json
+{
+  "code": 429,
+  "status": "ratelimited"
+}
+```
+
+##### Server Error (500)
+
+Implementations should of course be prepared to handle situations where a server is non-responsive for any reason and
+be prepared to handle any other, non-specified error codes including 500 codes.
+
+## Rationale: How does this CIP achieve its goals?
```

Comment:
For me; it makes more sense to move the explanation of how the alternative approach of paper wallets is not ideal to the `Rationale`.

Reply:
Done :)